Makefile

```diff
@@ -1,14 +1,25 @@
 
```

Comment:
Im not sure we will keep the make-file for the final release. I think it would be better if we just add another make-target specifically for osx. Then we don't need to deal with mixing platform specific stuff inside a single target and we can switch things around easier as we find out things.